package.json

```diff
@@ -53,14 +53,11 @@
         "prettier": "2.2.0",
         "react": "18.2.0",
         "react-native": "0.71.2",
-        "react-native-webview": "11.26.1",
         "react-native-builder-bob": "0.17.1",
+        "react-native-webview": "11.26.1",
         "react-test-renderer": "18.1.0",
         "typescript": "4.8.4"
     },
-    "resolutions": {
```

Comment:
just curious: why it is not needed anymore?

Reply:
I was looking into why we added it in the first place, it was removed automatically by yarn and did not break anything.
It was added in this commit: https://github.com/DataDog/dd-sdk-reactnative/commit/f1e2cf305b1ff948dcdde9fb4abbb0fd32349fc6
The goal at the time was to ensure we used the same types for react-native in the SDK and in the example app, even though versions were not aligned.

Now we keep them in sync, so there's no need for it anymore